src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/MsiInstallerStrings_de.wxl

```diff
@@ -1,5 +1,5 @@
 <?xml version = '1.0' encoding = 'utf-8'?>
-<WixLocalization Culture="de-de" xmlns="http://schemas.microsoft.com/wix/2006/localization" Codepage="1252">
+<WixLocalization Culture="de" xmlns="http://schemas.microsoft.com/wix/2006/localization" Codepage="1252">

```

Comment:
Why "de-de" was changed to "de"?

Reply:
Also not sure why this was changed in the translation. I'm reverting all instances of these language codes.

Reply:
This has been reverted as it was a regression for all locales of MsiInstallerStrings.wxl